app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -8,51 +8,81 @@
       // Indicate that js has worked
       $element.addClass('js-accordion-with-descriptions');
 
-       // Prevent FOUC, remove class hiding content
+      // Prevent FOUC, remove class hiding content
       $element.removeClass('js-hidden');
 
-      // Insert the markup for the subsection-controls
-      $element.prepend( '<div class="subsection-controls js-subsection-controls"><button aria-expanded="false">Open all</button></div>' );
+      var $subsectionButton = $element.find('.subsection__button');
+      var $subsectionHeader = $element.find('.subsection__header');
+      var totalSubsections = $element.find('.subsection__content').length;
 
-      var $subsectionTitle = $element.find('.subsection__title');
+      var $openOrCloseAllButton;
+      var GOVUKServiceManualTopic;
 
-      // Wrap each title in a button, with aria controls matching the ID of the subsection
-      $subsectionTitle.each(function(index) {
-        $(this).wrapInner( '<button class="subsection__button" aria-expanded="false" aria-controls="subsection_content_'+index+'"></button>' );
-      });
+      initialize();
 
-      var $subsectionButton = $element.find('.subsection__button');
-      var $subsectionHeader = $element.find('.subsection__header');
+      function initialize() {
+        addOpenCloseAllButton();
+        addSubsectionButtons();
+        addSubsectionIcons();
+        updateSubsectionControls();
+        getServiceManualTopicPrefix();
```

Comment:
`getServiceManualTopicPrefix()` just returns a value so there shouldn't be a need to call it here.